en/docs/security-reporting/reward-and-acknowledgement-program/index.md

```diff
@@ -77,13 +74,13 @@ We review reported security issues case-by-case. Following are common issues tha
 To show our appreciation, we provide a reward and an acknowledgement to eligible reporters after the reported issues are fixed and announced to the WSO2 customers and the community users.
 
 !!! tip
-    Please refer to our [Vulnerability Management Process]({{#base_path#}}/security-processes-programs/vulnerability-management-process/) for more details about how we disclose security vulnerabilities.
+    Please refer to our [Vulnerability Management Process](../../security-processes/vulnerability-management-process.md) for more details about how we disclose security vulnerabilities.
```

Comment:
```suggestion
    See our [Vulnerability Management Process](../../security-processes/vulnerability-management-process.md) for more details about how we disclose security vulnerabilities.
```

Reply:
Fixed in 4cf2f78e9fde97193b615b7bda53de23d579955c